roles/dotfiles/tasks/main.yml

```diff
@@ -5,23 +5,12 @@
     dest: ~/.oh-my-zsh
     force: yes
 
-- name: Clone syntax-highlighting plugin
+- name: Clone custom plugin
   git:
-    repo: https://github.com/zsh-users/zsh-syntax-highlighting.git
-    dest: ~/.oh-my-zsh/custom/plugins/zsh-syntax-highlighting
-    force: yes
-
-- name: Clone zsh-autosuggestions plugin
-  git:
-    repo: https://github.com/zsh-users/zsh-autosuggestions
-    dest: ~/.oh-my-zsh/custom/plugins/zsh-autosuggestions
-    force: yes
-
-- name: Clone symfony plugin
-  git:
-    repo: https://github.com/TheGrowingPlant/symfony.plugin.zsh.git
-    dest: ~/.oh-my-zsh/custom/plugins/symfony
+    repo: "{{ item.url }}"
+    dest: "~/.oh-my-zsh/custom/plugins/{{ item.name }}"
```

Comment:
I don't know ansible best practices but should we clean the dest dir before cloning plugins?